vaadin-grid-pro-flow/src/main/resources/META-INF/resources/frontend/editColumnConnector.js

```diff
@@ -0,0 +1,18 @@
+window.Vaadin.Flow.editColumnConnector = {
```

Comment:
I guess we need to add this kind of ugly boilerplate for each client resource to support OSGi:
https://github.com/vaadin/vaadin-combo-box-flow/blob/master/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/osgi/ComboBoxConnectorResource.java

Reply:
Similar resource is created